src/main/java/bio/terra/service/snapshotbuilder/query/Concept.java

```diff
@@ -0,0 +1,44 @@
+package bio.terra.service.snapshotbuilder.query;
+
+public class Concept extends TableVariable {
+
+  public static final String TABLE_NAME = "concept";
+  public static final String DOMAIN_ID = "domain_id";
+  public static final String CONCEPT_NAME = "concept_name";
+  public static final String CONCEPT_ID = "concept_id";
+  public static final String STANDARD_CONCEPT = "standard_concept";
+  public static final String CONCEPT_CODE = "concept_code";
+  private static final TablePointer tablePointer = TablePointer.fromTableName(TABLE_NAME);
+
+  public Concept() {
+    super(tablePointer, null, null, false);
+  }
+
+  public Concept(TableVariableBuilder tableVariableBuilder) {
+    super(
+        tablePointer,
+        tableVariableBuilder.getJoinField(),
+        tableVariableBuilder.getJoinFieldOnParent(),
+        tableVariableBuilder.isLeftJoin());
+  }
+
+  public FieldVariable name() {
+    return this.makeFieldVariable(CONCEPT_NAME);
```

Comment:
It might not be necessary, but I think it would be better to re-use fields instead of making a new one each time. Using a `Map` as a cache would work well something like
```
    private Map<String, FieldVariable> fields = new HashMap<>();

  ....

    return fields.computeIfAbsent(CONCEPT_NAME, this::makeFieldVariable);
```

Reply:
I like this idea! and it doesn't add much complexity